vm/vmArraysAndMaps_test.go

```diff
@@ -20,981 +20,992 @@ var (
-		{script: "make([]string, 0)", runOutput: []string{}},
-
-		{script: "make([]bool, 2)", runOutput: []bool{false, false}},
-		{script: "make([]int32, 2)", runOutput: []int32{int32(0), int32(0)}},
-		{script: "make([]int64, 2)", runOutput: []int64{int64(0), int64(0)}},
-		{script: "make([]float32, 2)", runOutput: []float32{float32(0), float32(0)}},
-		{script: "make([]float64, 2)", runOutput: []float64{float64(0), float64(0)}},
-		{script: "make([]string, 2)", runOutput: []string{"", ""}},
-
-		{script: "make([]bool, 0, 2)", runOutput: []bool{}},
-		{script: "make([]int32, 0, 2)", runOutput: []int32{}},
-		{script: "make([]int64, 0, 2)", runOutput: []int64{}},
-		{script: "make([]float32, 0, 2)", runOutput: []float32{}},
-		{script: "make([]float64, 0, 2)", runOutput: []float64{}},
-		{script: "make([]string, 0, 2)", runOutput: []string{}},
-
-		{script: "make([]bool, 2, 2)", runOutput: []bool{false, false}},
-		{script: "make([]int32, 2, 2)", runOutput: []int32{int32(0), int32(0)}},
-		{script: "make([]int64, 2, 2)", runOutput: []int64{int64(0), int64(0)}},
-		{script: "make([]float32, 2, 2)", runOutput: []float32{float32(0), float32(0)}},
-		{script: "make([]float64, 2, 2)", runOutput: []float64{float64(0), float64(0)}},
-		{script: "make([]string, 2, 2)", runOutput: []string{"", ""}},
-
-		{script: "a = make([]bool, 0); a += true; a += false", runOutput: []bool{true, false}, output: map[string]interface{}{"a": []bool{true, false}}},
-		{script: "a = make([]int32, 0); a += 1; a += 2", runOutput: []int32{int32(1), int32(2)}, output: map[string]interface{}{"a": []int32{int32(1), int32(2)}}},
-		{script: "a = make([]int64, 0); a += 1; a += 2", runOutput: []int64{int64(1), int64(2)}, output: map[string]interface{}{"a": []int64{int64(1), int64(2)}}},
-		{script: "a = make([]float32, 0); a += 1.1; a += 2.2", runOutput: []float32{float32(1.1), float32(2.2)}, output: map[string]interface{}{"a": []float32{float32(1.1), float32(2.2)}}},
-		{script: "a = make([]float64, 0); a += 1.1; a += 2.2", runOutput: []float64{float64(1.1), float64(2.2)}, output: map[string]interface{}{"a": []float64{float64(1.1), float64(2.2)}}},
-		{script: "a = make([]string, 0); a += \"a\"; a += \"b\"", runOutput: []string{"a", "b"}, output: map[string]interface{}{"a": []string{"a", "b"}}},
-
-		{script: "a = make([]bool, 2); a[0] = true; a[1] = false", runOutput: false, output: map[string]interface{}{"a": []bool{true, false}}},
-		{script: "a = make([]int32, 2); a[0] = 1; a[1] = 2", runOutput: int64(2), output: map[string]interface{}{"a": []int32{int32(1), int32(2)}}},
-		{script: "a = make([]int64, 2); a[0] = 1; a[1] = 2", runOutput: int64(2), output: map[string]interface{}{"a": []int64{int64(1), int64(2)}}},
-		{script: "a = make([]float32, 2); a[0] = 1.1; a[1] = 2.2", runOutput: float64(2.2), output: map[string]interface{}{"a": []float32{float32(1.1), float32(2.2)}}},
-		{script: "a = make([]float64, 2); a[0] = 1.1; a[1] = 2.2", ru
```

Comment:
Please add more tests to get lines 740 & 745 in vmExpr.go.
Tests like delete(1++, "x") & delete(a, 1++)

Reply:
done.